app/admin/dashboard.rb

```diff
@@ -1,6 +1,15 @@
 ActiveAdmin.register_page 'Dashboard' do
   menu priority: 1, label: proc { I18n.t('active_admin.dashboard') }
 
+  page_action :reseed, method: :post do
+    Lessons::ImportAllContentJob.perform_async
+    redirect_to admin_dashboard_path, notice: 'Curriculum lesson import job running 🐢 Track the progress at /sidekiq'
+  end
+
+  action_item :reseed do
+    link_to 'Seed Curriculum', admin_dashboard_reseed_path, method: :post
+  end
+
```

Comment:
Do we want a spec for this? Was unsure where to put it as there's only system tests for admin.

Reply:
I think no spec for this is fine, the effort vs value of having a spec for this one doesn't make it worth it imo.